plugin/arcconf/arcconf.py

```diff
@@ -1,5 +1,5 @@
 # Copyright (C) 2015-2016 Red Hat, Inc.
-# (C) Copyright 2015-2016 Hewlett Packard Enterprise Development LP
```

Comment:
@brraghav You should not do that. If you copied code from hpsa plugin, you should keep the original license line and only add your new one.